configs/scannetpp/semseg-spunet-v1m1-0-base.py

```diff
@@ -20,7 +20,10 @@
         channels=(32, 64, 128, 256, 256, 128, 96, 96),
         layers=(2, 3, 4, 6, 2, 2, 2, 2),
     ),
-    criteria=[dict(type="CrossEntropyLoss", loss_weight=1.0, ignore_index=-1)],
+    criteria=[
+        dict(type="CrossEntropyLoss", loss_weight=1.0, ignore_index=-1),
+        dict(type="LovaszLoss", mode="multiclass", loss_weight=1.0, ignore_index=-1),
```

Comment:
LovaszLoss added in the criteria here to be consistent with our other experiments. Let me know if this config should be in a seperate file. i.e `configs/scannetpp/semseg-spunet-v1m1-0-lovasz.py`.

Reply:
Yeah, I will put this change in another file, as this is a new setting different from the original paper. For base config, let's respect the setting from original paper. (I will make the modification)